.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+Thanks for submitting an issue! 
+
+Please answer these questions
+
+-------------ANY REQUEST NEEDS THE FOLLOWING --------------------
+
+I. What keywords did you search in Kubernetes issues before filing this one? (If you have found any duplicates, you should instead reply there.):
+
+II. Is this a BUG REPORT or FEATURE REQUEST- fill out appropriate section below
+
+
+-------------FEATURE REQUEST --------------------
+
+A. Describe IN DETAIL the feature/behavior/change you'd like to see.
+
+In both cases, be ready for followup questions, and please respond in a timely
+manner.  If we can't reproduce a bug or think a feature already exists, we
+might close your issue.  If we're wrong, PLEASE feel free to reopen it and
+explain why.
+
+B. Feel free to submit design documentation supporting your feature request; submit a link which is publically avaliable. 
+
+-------------BUG REPORT --------------------
+1. Fill in as much of the template below as you can.  If you leave out
+    information, we can't help you as well.
+
+2. What Kubernetes version are you running? use `kubectl version`
+
+3. What Environment are you running on?
+- **Cloud provider or hardware configuration**:
+- **OS** (e.g. from /etc/os-release):
+- **Kernel** (e.g. `uname -a`):
+- **Install tools**:
+- **Others**:
+
+4. What commands did you execute (Please provide screenshot) and what happened after commands executed?
```

Comment:
I would leave out the screenshot request, that's asking a lot for our users. Suggest

```
4. What commands did you execute (Screenshot if available) and what happened after commands executed?
```